provider/cloudflare_test.go

```diff
@@ -336,9 +336,42 @@ func (m *mockCloudFlareUpdateRecordsFail) ListZones(zoneID ...string) ([]cloudfl
 }
 
 func TestNewCloudFlareChanges(t *testing.T) {
-	action := cloudFlareCreate
 	endpoints := []*endpoint.Endpoint{{DNSName: "new", Target: "target"}}
-	_ = newCloudFlareChanges(action, endpoints, true)
+	newCloudFlareChanges(cloudFlareCreate, endpoints, true)
+}
+
+func TestNewCloudFlareChang(t *testing.T) {
+	change := newCloudFlareChange(cloudFlareCreate, &endpoint.Endpoint{DNSName: "new", RecordType: "A", Target: "target"}, false)
+	assert.False(t, change.ResourceRecordSet.Proxied)
+}
+
+var cloudFlareTypes = []struct {
```

Comment:
Since it's only used in TestNewCloudFlareChangeProxiable, could you please move it in the function?

Reply:
Otherwise LGTM